packages/storage/src/reference.ts

```diff
@@ -118,7 +120,39 @@ export class Reference {
 }
 
 /**
- * Uploads a blob to this object's location.
+ * Uploads data to this object's location.
+ * The upload is not resumable.
+ * @public
+ * @param ref - Storage Reference where data should be uploaded.
```

Comment:
"Storage reference" or "StorageReference" ?

Reply:
Hmm, exported type name is just `Reference`. I guess I should rename it since it's publicly exposed. Compat stays the same since it's namespaced and to remain compat.